container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -2102,6 +2199,57 @@ configuration:
     default: support@example.com
     description: Support contact information for the cluster
     required: true
+  - name: SYSLOG_ADAPT_CERT
+    secret: true
+    generator:
+      id: syslog_adapt_cert
+      type: Certificate
+      value_type: certificate
```

Comment:
You should have a `subject_names` list of "Subject Alternate Names" for the cert and/or a `role_name` that will be used to generate the various internal names with `KUBE_SERVICE_DOMAIN_SUFFIX` etc.

Reply:
Hm. Will check existing certs and keys for examples. Sort of like the `cn` fields which got added to the `opinions.yml` for this, I guess ?

Reply:
See commit 84c47e4c94470197738340ded7aadd7453b71f56.
Note, my names might be wrong.
As nothing changes. I.e. roles report same issue as before, with validation.